temba/utils/whatsapp/tasks.py

```diff
@@ -181,3 +183,94 @@ def refresh_whatsapp_templates():
 
             except Exception as e:
                 logger.error(f"Error refreshing whatsapp templates: {str(e)}", exc_info=True)
+
+
+def update_is_active_catalog(channel, catalogs_data):
+    waba_id = channel.config.get("wa_waba_id", None)
+
+    if not waba_id:
+        raise ValueError("Channel wa_waba_id not found")
+
+    url = f"https://graph.facebook.com/v17.0/{waba_id}/product_catalogs"
```

Comment:
Some code snippets you are using version 16, others 17, make sure it is in the right version, the ideal is to keep everything in one version.